templates/_macros/split-list.jinja

```diff
@@ -0,0 +1,91 @@
+# Params
+# is_description_full_width_on_desktop: whether description has its own row on
+#   desktop vs. 50/50 split between title/description
+# is_list_full_width_on_tablet: whether list title/description each have their
+#   own row on tablet vs. 50/50 split between title/description
+{% macro split_list(
+  is_description_full_width_on_desktop=true,
+  is_list_full_width_on_tablet=true) -%}
+  {% set title_content = caller('title') %}
+  {% set description_content = caller('description') %}
+  {% set has_description = description_content|trim|length > 0 %}
+
+  <div class="u-fixed-width">
+    <hr class="u-hide--small">
+
+    {% if has_description == true %}
+      {% if is_description_full_width_on_desktop == true %}
+        <div class="row">
+          <div class="col-12">
+            {{ title_content }}
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="col-6 col-start-large-7">
+            {{ description_content }}
+          </div>
+        </div>
+      {% else %}
+        <div class="row">
+          <div class="col-6">
+            {{ title_content }}
+          </div>
+
+          <div class="col-6">
+            {{ description_content }}
+          </div>
+        </div>
+      {% endif %}
+    {% else %}
+      <div class="row">
+        <div class="col-12">
+          {{ title_content }}
+        </div>
+      </div>
+    {% endif %}
+
+    {% for number in range(1, 10) %}
```

Comment:
I wonder if we should alert the user in some way if they pass in more list items than are allowed - such as with a console error? Or should we simply rely on them to notice when they are missing content they expect to see?

Reply:
We need to verify with Lyubo if there is an upper limit.

I guess it doesn't hurt us too much if we increase it up to 50 or 100 (which should be way more than enough?)